request.go

```diff
@@ -169,6 +170,13 @@ func (r *gsRequest) prepareHTTPRequest(ctx context.Context, cfg *Config) (*http.
 		request.Header.Set(k, v)
 	}
 
+	// Set query parameters if there are any of them.
+	query := request.URL.Query()
+	for k, v := range r.queryParameters {
+		query.Add(k, v)
```

Comment:
should not we validate the query parameters? Or we leave the validation to API. What are the expected query parameters?

Reply:
this is not exported, so it can be used only by the lib only. The query parameters depend on the API endpoint. E.g. usage endpoints have `from_time`, `without_deleted` and so on.